cmd/ttn-lw-cli/commands/simulate.go

```diff
@@ -502,8 +514,9 @@ var (
 	}
 
 	simulateDataUplinkCommand = &cobra.Command{
-		Use:   "uplink",
-		Short: "Simulate a data uplink (EXPERIMENTAL)",
+		Use:    "gateway-uplink",
+		Short:  "Simulate a gateway uplink message (EXPERIMENTAL)",
```

Comment:
I think the description should be more clear about the fact that this "simulates an uplink message from an end device, sent through a simulated gateway connection". I think "gateway uplink message" alone is not clear enough.